### PR TITLE
Trino Guru added on Add on

### PR DESCRIPTION
**Pull request by kursataktas**
Details can be found on this [PR](https://github.com/trinodb/trino/pull/24210)

### CODE REVIEW

**Comment by kursataktas** [comment]
Email sent

File: _data/tools.yml [hunk 1]
```diff
@@ -1697,3 +1697,15 @@
       url: https://docs.getzingdata.com/docs/setting-up-a-data-source/trino_setup/
     - urltext: Zing Data with Trino tutorial
       url: https://docs.starburst.io/data-consumer/clients/zingdata.html
+- name: Trino Guru
```
Comment (mosabua) [inline _data/tools.yml:1700]:
```suggestion
- name: Guru Base
```

to avoid using the Trino brand

File: _data/tools.yml [hunk 2]
```diff
@@ -1697,3 +1697,15 @@
...
+  anchor: trino-guru
```
Comment (mosabua) [inline _data/tools.yml:1701]:
```suggestion
  anchor: guru-base
```

File: _data/tools.yml [hunk 3]
```diff
@@ -1697,3 +1697,15 @@
...
+  category: add-on
+  logo: /assets/images/logos/gurubase_logo.png
```
Comment (mosabua) [inline _data/tools.yml:1703]:
```suggestion
  logo: /assets/images/logos/guru-base.png
```

File: _data/tools.yml [hunk 4]
```diff
@@ -1697,3 +1697,15 @@
...
+  logosmall: /assets/images/logos/gurubase_logo_small.png
```
Comment (mosabua) [inline _data/tools.yml:1704]:
```suggestion
  logosmall: /assets/images/logos/guru-base-small.png
```

Please rename file accordingly

File: _data/tools.yml [hunk 5]
```diff
@@ -1697,3 +1697,15 @@
...
+  description: |
+    Trino Guru is a Trino-focused AI to answer your questions.
```
Comment (mosabua) [inline _data/tools.yml:1706]:
```suggestion
    Guru Base is an AI assistant to answer your questions. It includes knowledge about   Trino and 
```

File: _data/tools.yml [hunk 6]
```diff
@@ -1697,3 +1697,15 @@
...
     - urltext: Zing Data with Trino tutorial
       url: https://docs.starburst.io/data-consumer/clients/zingdata.html
+- name: Trino Guru
+  anchor: trino-guru
+  category: add-on
+  logo: /assets/images/logos/gurubase_logo.png
+  logosmall: /assets/images/logos/gurubase_logo_small.png
+  description: |
+    Trino Guru is a Trino-focused AI to answer your questions. 
+    It uses the data from the Trino repository and data from the official
```
Comment (mosabua) [inline _data/tools.yml:1707]:
```suggestion
    uses the data from the Trino source code the
```

File: _data/tools.yml [hunk 7]
```diff
@@ -1697,3 +1697,15 @@
...
+    Trino documentation to generate answers by leveraging the LLM.
```
Comment (mosabua) [inline _data/tools.yml:1708]:
```suggestion
   documentation to generate answers by leveraging the LLM. 
```

**Review by mosabua — CHANGES_REQUESTED** [review]
Please apply suggestions, rename the images, and resize them to 500x400 and 250x200.

File: _data/tools.yml (hunk 5 above)
Comment (kursataktas) [reply _data/tools.yml]: I changed the entire text and am open to feedback or review.

File: _data/tools.yml (hunk 6 above)
Comment (kursataktas) [reply _data/tools.yml]: To clarify, Gurubase does not index the codebase; it only indexes text-based content, such as READMEs in the repository and documentation pages.

File: _data/tools.yml (hunk 7 above)
Comment (kursataktas) [reply _data/tools.yml]: I changed the entire text and am open to feedback or review.

**Comment by kursataktas** [comment]
> Please apply suggestions, rename the images, and resize them to 500x400 and 250x200.

I updated the PR. The image sizes are the same as you mentioned, so I did not change them. Please let me know if I am mistaken.

File: _data/tools.yml (hunk 6 above)
Comment (mosabua) [reply _data/tools.yml]: Sounds good

File: _data/tools.yml (hunk 5 above)
Comment (mosabua) [reply _data/tools.yml]: ok

**Comment by mosabua** [comment]
Please squash commits and change commit to 

"Add Gurubase as add-on"

**Comment by mosabua** [comment]
We will wait for the CLA processing with merge.

File: _data/tools.yml [hunk 8]
```diff
@@ -1697,3 +1697,15 @@
       url: https://docs.getzingdata.com/docs/setting-up-a-data-source/trino_setup/
     - urltext: Zing Data with Trino tutorial
       url: https://docs.starburst.io/data-consumer/clients/zingdata.html
+- name: Gurubase
+  anchor: gurubase
+  category: add-on
+  logo: /assets/images/logos/gurubase.png
```
Comment (mosabua) [inline _data/tools.yml:1703]: please add

owner: other

**Comment by mosabua** [comment]
I rebased and fixed the ordering. Ready to go now.

**Comment by kursataktas** [comment]
Hello @mosabua,

We recently released the embeddable "Ask AI" widget, which allows you to integrate your Guru’s Q&A capability into websites, documentation, and more.

The installation process is straightforward, as explained in [this GitHub repository](https://github.com/getanteon/gurubase-widget).

You can see it in action on the [Anteon](https://getanteon.com/) and [NetAlertX](https://netalertx.com/) websites.

**Comment by kursataktas** [comment]
Hello @mosabua,

I want to share an exciting update about Gurubase. We've recently released several great features that I think you'll find interesting.

### 1. Slack & Discord Integration
You can now add your Guru as a bot to Slack and Discord, allowing you to ask questions instantly. More details here: [Discord Bot Integration](https://docs.gurubase.ai/integrations/discord-bot).

### 2. Analytics Dashboard
Track what your users ask your Guru and gain valuable insights. This feature is available on the "My Gurus" page. If you haven’t maintained your Guru yet, I highly recommend doing so to access your analytics. More information about claiming your Guru can be found [here](https://github.com/Gurubase/gurubase?tab=readme-ov-file#how-to-claim-a-guru).
image

### 3. API Support
All Guru functionalities are now accessible via API. You can ask questions, edit data sources, and more using our API endpoints. Here is the [API Doc](https://docs.gurubase.ai/api-reference/introduction).

### 4. Codebase Indexing
We now support indexing codebases from GitHub repositories to generate answers for related queries. This feature is available for select Gurus. If you don’t see the "Codebase Indexed" label on your Guru edit page and would like access, please let me know, I’d be happy to enable it for you!
image

Let me know if you have any questions or feedback!

**Pull request merged**